woocommerce/facebook-for-woocommerce
# regenerate package-lock.json to update indirect dependencies

### Changes proposed in this Pull Request:

This PR updates our package-lock file to update various indirect dependencies (dependencies of our dependencies - sake and `@wordpress/scripts`).

There shouldn't be any change to behaviour - our scripts are all essentially build tooling. 

### How to test the changes in this Pull Request:
1. Test a clean build - new clone, delete dependencies folders and/or `npm ci`.
2. Build a release build of plugin (to test critical path of build tooling) - `npm run build`.
2. Ensure plugin activates correctly.
3. Broad smoke testing of key flows - product sync works, messenger works, etc.

### Changelog entry
> Dev - Update various npm dependencies
